src/main/java/de/teamlapen/vampirism/entity/minions/vampire/VampireMinionBaseEntity.java

```diff
@@ -70,7 +70,6 @@ public void activateMinionCommand(IMinionCommand command) {
 
     @Override
     public boolean canAttack(EntityType<?> typeIn) {
-        //if(typeIn.equals(ModEntities.portal_guard) return false;
         return super.canAttack(typeIn);
```

Comment:
If you remove that comment, you can just remove the entire method :D

Reply:
just a test if you reading every line :)